src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -946,8 +945,10 @@ def create_vm(cmd, vm_name, resource_group_name, image=None, size='Standard_DS1_
     # In order to avoid breaking change which has a big impact to users,
     # we use the hint to guide users to use Standard public IP to create VM in the first stage.
     if cmd.cli_ctx.cloud.profile == 'latest':
-        if public_ip_sku is None and public_ip_address_type == 'new' or public_ip_sku == "Basic":
-            logger.warning(remove_basic_option_msg, "--public-ip-sku Standard", "Public IP")
+        if public_ip_sku is None and public_ip_address_type == 'new':
+            public_ip_sku = 'Standard'
```

Comment:
If this comment https://github.com/Azure/azure-cli/pull/27691/files#r1378560321 can be considered, then the code here is no longer needed, right?